demos/mediapipe/object_detection/README.md

```diff
@@ -28,6 +28,7 @@ python mediapipe_object_detection.py --download_models
 ## Server Deployment
 :::{dropdown} **Deploying with Docker**
 ```bash
+sed -i 's;ssdlite_object_detection_labelmap.txt;/demo/ssdlite_object_detection_labelmap.txt;g' graph.pbtxt
```

Comment:
Why do we need this? Can't we use relative path in docker?

Reply:
This is a built-in mediapipe mechanism not ovms.